src/mumble/JackAudio.h

```diff
@@ -100,6 +103,7 @@ class JackAudioSystem : public QObject {
 	bool unregisterPort(jack_port_t *port);
 	bool connectPort(jack_port_t *sourcePort, jack_port_t *destinationPort);
 	bool disconnectPort(jack_port_t *port);
+	bool renamePort(jack_port_t *port, const char *name);
```

Comment:
Is `nullptr` a valid argument for the arguments? Otherwise, references should be used (`const std::string &` in the second case, unless you exclusively pass String literals into the function)

Reply:
`nullptr` is a valid argument to the raw `jack_port_rename` routine both for `port` and `name`, though it will return an error value in either case.